api/client/container/stop.go

```diff
@@ -40,16 +41,35 @@ func NewStopCommand(dockerCli *client.DockerCli) *cobra.Command {
 
 func runStop(dockerCli *client.DockerCli, opts *stopOptions) error {
 	ctx := context.Background()
+	timeout := time.Duration(opts.time) * time.Second
 
+	var waitAll sync.WaitGroup
+	var printLock sync.Mutex
+	var errLock sync.Mutex
 	var errs []string
+
+	var maxParallel int32 = 1000
```

Comment:
By "timeout" I meant that we could do bigger numbers if we detected that all current stop calls were sleeping between sigterm and sigkill. I'm not suggesting we'd do that - lowering the number to 20 or 50 should be good enough.